packages/salesforcedx-vscode-replay-debugger/src/core/logContext.ts

```diff
@@ -113,7 +115,14 @@ export class LogContext {
     const typerefMapping = this.breakpointUtil.getTyperefMapping();
     let uri = '';
     typerefMapping.forEach((value, key) => {
-      const processedKey = key.replace('/', '.').replace('$', '.');
+
+      let processedKey = "";
```

Comment:
Use single quotes.

Reply:
Doh! I fixed this locally, I'll get a new update shortly.